widgetastic_manageiq/__init__.py

```diff
@@ -394,18 +394,20 @@ def _values_to_add(self, values):
         return list(set(values) - set(self.selected_text))
 
     def fill(self, values):
-        if set(values) == set(self.selected_text):
-            return False
+        if values:
```

Comment:
your implementation looks good. but I cannot get why we pass only switch names and change their values to opposite.

I guess we should pass list of switches and their values instead. But this stays apart of this PR

Reply:
Hmm... yeah, I'll think about this for a future PR. I think initially this was done because the switch only has either on or off, so passing it's value is kind of unnecessary.